lib/rules/valid-v-bind-sync.js

```diff
@@ -82,6 +82,22 @@ function maybeNullObjectMemberExpression(node) {
   return false
 }
 
+function isValidIs(node) {
+  const { attributes } = node
+  const isAttribute = attributes.some((attr) => {
+    // check for `VAttribute`
+    if (attr.type === 'VAttribute') {
+      // check for `is` attribute
+      if (attr.key.type === 'VIdentifier' && attr.key.name === 'is') return true
+
+      // check for `:is` `bind` attribute
+      if (attr.key.type === 'VDirectiveKey' && attr.key.argument.name === 'is')
```

Comment:
You need to check if it is v-bind. Also, the argument can be null.

https://github.com/vuejs/vue-eslint-parser/blob/master/docs/ast.md#vdirectivekey

In addition, could you add a test to make sure it is not judged as `is`?

```vue
<tr v-on:is="myRow" :some-prop.sync="somePropValue">
<tr v-bind="myRow" :some-prop.sync="somePropValue">
```